Cartfile

```diff
@@ -1,4 +1,4 @@
-binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" == 23.2.0-rc.3
-binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.xcframework.json" ~> 122.0
+binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" ~> 23.2.1
```

Comment:
As I can see, [Maps depends on Common 23.2.1.](https://github.com/CocoaPods/Specs/blob/master/Specs/8/a/8/MapboxMaps/10.10.0/MapboxMaps.podspec.json#L37)
Should we then change the expected Common version in the release schedule from 23.2.0 to 23.2.1?